fr.css

```diff
@@ -0,0 +1,530 @@
+/*Terms of Service link*/
+.swagger-ui .info__tos a {
+    visibility: hidden;
+}
+.swagger-ui .info__tos a::before {
+    visibility: visible;
+    content: "Mandat";
+}
+
+/*Open Government License - Canada link */
+.swagger-ui .info__license a {
+    visibility: hidden;
+}
+.swagger-ui .info__license a::before {
+    visibility: visible;
+    content: "Licence du gouvernement ouvert - Canada";
```

Comment:
this one is specific to our API so we won't be able to use it with ckanext-openapiview

Reply:
Why not split the license title in the yaml source instead?

Reply:
Good to know, we won't copy this over to the API extension.

Reply:
A better fix would be to split the license title in the yaml source.